app/src/features/rules/screens/rulesList/components/RulesList/components/GettingStarted/gettingStarted.scss

```diff
@@ -0,0 +1,182 @@
+
+            .caption {
+              font-size: 15px;
+              font-weight: 400;
+              line-height: 150%; /* 22.5px */
+              color: var(--requestly-color-text-subtle, #bbb);
+            }
+          }
+        }
+
+        .rule-suggestions {
+          display: flex;
+          flex-direction: column;
+          align-items: flex-start;
+          gap: 12px;
+          flex: 1 0 0;
+
+          .suggested-rule-link {
+            padding: 16px;
+            display: flex;
+            align-items: center;
+            gap: 8px;
+            flex-wrap: nowrap;
+            border-radius: 8px;
+            width: 100%;
+            cursor: pointer;
+            transition: all 0.2s;
+            color: var(--requestly-color-text-subtle, #bbb);
+            background: var(--requestly-color-surface-0, #212121);
+
+            &:hover {
+              background: var(--requestly-color-surface-1, #282828);
+            }
+
+            .icon {
+              width: 20px;
+              height: 20px;
+              flex-shrink: 0;
+
+              .rule-type-icon {
+                padding: 2px;
+              }
+            }
+
+            .title {
+              font-size: 15px;
+              font-weight: 400;
+              line-height: 150%;
+              font-size: 15px;
+              font-weight: 400;
+              line-height: 150%; /* 22.5px */
+              color: var(--requestly-color-text-subtle, #bbb);
+            }
+
+            .help-icon {
+              margin-left: auto;
+              font-size: 16px;
+              line-height: 16px; /* 100% */
+              opacity: 0.5;
+              color: var(--requestly-color-text-subtle, #bbb);
+            }
+          }
+        }
+      }
+    }
+
+    .divider.ant-divider {
+      margin-top: 32px;
+      margin-bottom: 16px;
+    }
+
+    .getting-started-actions {
+      display: flex;
+      align-items: center;
+      gap: 32px;
+
+      .templates-btn {
+        margin-left: auto;
+      }
+    }
+
+    .ask-ai-container {
+      display: flex;
+      flex-direction: column;
+      gap: 8px;
+      align-self: stretch;
+      margin-top: 56px;
+
+      .title {
+        font-size: 15px;
+        font-weight: 500;
+        line-height: 150%; /* 22.5px */
+        color: var(--requestly-color-text-default, #fff);
+      }
+
+      .ask-ai {
+        height: 48px;
+        padding: 16px;
+        display: flex;
+        align-items: center;
+        gap: 8px;
+        align-self: stretch;
+        font-size: 13px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 150%;
+        border-radius: 8px;
+        border: 1px solid rgba(255, 255, 255, 0.4);
+        color: var(--requestly-color-text-placeholder, #8f8f8f);
+
+        backdrop-filter: blur(12px);
+        box-shadow: 0px 4px 8px 0px rgba(0, 0, 0, 0.15);
+        background: var(--requestly-color-white-t-10, rgba(255, 255, 255, 0.06));
+      }
+
+      .caption {
```

Comment:
Does the style not conflict with the `caption` class defined in`base.css`?
Can we give this a different name instead?

Reply:
No it won't since its a `scss` file and inside `.getting-started-container` class